docs/skills-client/js.md

```diff
@@ -73,6 +74,29 @@ Alternatively you can import individually:
 </head>
  ``` 
 
+## Option 3: ```<script type="text/javascript">``` tag
+
+You may also import import the module using the html ```<script type="text/javascript">``` tag in order to support older
```

Comment:
remove doubled word (import). Change "in order to support older browser" to browers or "an older browser"